### ISSUE TITLE
Accessibility issues with multi-select dropdown

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shiny-template#121]
**Describe the bug**
Flagged in testing as a WCAG failing as the multi-select was problematic for screen reader users as the expected behaviour is not intuitive and elements have not been created using the correct mark-up.

**Screenshots**
![Image](https://github.com/user-attachments/assets/f06158df-e85a-459c-855a-cb808ce4a383)

**Additional context**

From the design system - https://design-system.service.gov.uk/components/select/

Avoid adding functionality to allow selecting multiple options. The select component does not support selecting multiple options, as there’s a history of poor usability and assistive technology support for `select multiple` - https://www.24a11y.com/2019/select-your-poison 

If you need to ask the user to pick more than one item from a list, it’s almost always better to use another method, such as a list of checkboxes. 

WCAG 2.2 - Any input that’s designed to let a user select multiple options must offer a way to do so without relying on ‘click and drag’ movements or keyboard and mouse combination actions. 

This is to comply with [WCAG 2.2 success criterion 2.5.7 Dragging movements](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html).

**Suggestion**

We look into alternatives, such as checkbox lists to replace this kind of multi-select
